feature.env

```diff
@@ -15,4 +15,4 @@ CONFIRM_ENABLED=true
 CONFIRM_NO_IDENTIFIERS_ENABLED=true
 TEMPORARY_ABSENCE_ENABLED=true
 CONFIRM_COURT_RETURN_ENABLED=true
-SUPPORTING_MULTITRANSACTIONS_ENABLED=true
+SUPPORTING_MULTITRANSACTIONS_ENABLED=false
```

Comment:
NBD but it would possibly be easier to change the integration tests to work with the flag enabled - this means we don't have to fix the tests when we remove the flag... we can leave as is though

Reply:
Ah cool, just wasn't sure - I'll do that quickly now